src/build-disk-image.ts

```diff
@@ -22,12 +22,25 @@ import * as os from 'node:os';
 import * as fs from 'node:fs';
 import { resolve } from 'node:path';
 import * as containerUtils from './container-utils';
+import * as machineUtils from './machine-utils';
 import { bootcImageBuilderContainerName, bootcImageBuilderName } from './constants';
 import type { History } from './history';
 
 const telemetryLogger = extensionApi.env.createTelemetryLogger();
 
 export async function buildDiskImage(imageData: unknown, history: History) {
+  // Before we do ANYTHING, we should be checking to see if the podman machine is rootful or not
+  // as that's a requirement for bootc-image-builder to work correctly.
+  const isRootful = await machineUtils.isPodmanMachineRootful();
+  if (!isRootful) {
+    await extensionApi.window.showErrorMessage(
+      'The podman machine is not set as rootful. Please recreate the podman machine with rootful privileges set.',
+      'OK',
+      'Cancel',
```

Comment:
Two buttons that do the same thing. :)

Remove the Cancel button and end the message with "... and try again."?

Reply:
Fixed.